[Docs] Add overview page for visual-formatting (#742)
mckinsey/vizro: vizro-core/docs/pages/user-guides/visual-formatting.md

# How to customize the style of Vizro dashboards

> Vizro comes with a default styling aimed at helping users with no design experience get started quickly.

Reviewer: ```suggestion
Vizro has a default styling to help users with no design experience get started quickly.
```

> However, if you wish to customize further, there are several options available:

Reviewer: ```suggestion
There are several options if you want to customize the style:
```
